app/src/main/java/fr/free/nrw/commons/contributions/Contribution.kt

```diff
@@ -112,5 +112,6 @@ data class Contribution constructor(
         fun formatDescriptions(descriptions: List<UploadMediaDetail>) =
             descriptions.filter { it.descriptionText.isNotEmpty() }
                 .joinToString { "{{${it.languageCode}|1=${it.descriptionText}}}" }
+                .replace("}}, ", "}}")
```

Comment:
Would not it be better to not add the comma in the first place?

Reply:
I tried that first but I was having a hard time determining where the comma was actually originating from, I'll dig a little more until I can find it. 

Thanks for the quick response, I apologize if I'm making you work more than needed.

Reply:
Okay, I think this one should do it, I should've checked the default separator for the .joinToString first before trying to look for the comma everywhere else.